tests/include/tc_util.h

```diff
@@ -67,22 +67,19 @@
 	} while (0)
 
 #define TC_PRINT(fmt, ...) PRINT_DATA(fmt, ##__VA_ARGS__)
-#define TC_START(name) PRINT_DATA("tc_start() - %s\n", name)
+#define TC_START(name) PRINT_DATA("start() - %s\n", name)
 #define TC_END(result, fmt, ...) PRINT_DATA(fmt, ##__VA_ARGS__)
 
 /* prints result and the function name */
 #define _TC_END_RESULT(result, func)					\
-	do {								\
-		PRINT_LINE;						\
 		TC_END(result, "%s - %s.\n",				\
-		       (result) == TC_PASS ? PASS : FAIL, func);	\
-	} while (0)
+		       (result) == TC_PASS ? PASS : FAIL, func)
+
 #define TC_END_RESULT(result)                           \
 	_TC_END_RESULT((result), __func__)
 
 #define TC_END_REPORT(result)                               \
 	do {                                                    \
-		PRINT_LINE;                                         \
```

Comment:
looking at the output with this change and what we had before, maybe the solution was to fix how we use the lines instead of just removing them, we had:
```
***** BOOTING ZEPHYR OS v1.8.99 - BUILD: Aug 15 2017 19:16:37 *****
Running test suite test_sema_api
tc_start() - test_sema_thread2thread
===================================================================
PASS - test_sema_thread2thread.
tc_start() - test_sema_thread2isr
===================================================================
PASS - test_sema_thread2isr.
tc_start() - test_sema_reset
===================================================================
PASS - test_sema_reset.
tc_start() - test_sema_count_get
===================================================================
PASS - test_sema_count_get.
===================================================================
PROJECT EXECUTION SUCCESSFUL
```

note how the lines were in the wrong position? it should be:

```
***** BOOTING ZEPHYR OS v1.8.99 - BUILD: Aug 15 2017 19:16:37 *****
Running test suite test_sema_api
===================================================================
tc_start() - test_sema_thread2thread
PASS - test_sema_thread2thread.
===================================================================
tc_start() - test_sema_thread2isr
PASS - test_sema_thread2isr.
===================================================================
tc_start() - test_sema_reset
PASS - test_sema_reset.
===================================================================
tc_start() - test_sema_count_get
PASS - test_sema_count_get.
===================================================================
PROJECT EXECUTION SUCCESSFUL
```
That already looks much better, and if logging is enabled, the log messages would be enclosed within the same lines of the same test.